token-metadata/interface/src/instruction.rs

```diff
@@ -34,80 +36,47 @@ pub struct Initialize {
     /// URI pointing to more metadata (image, video, etc.)
     pub uri: String,
 }
-impl TlvDiscriminator for Initialize {
-    /// Please use this discriminator in your program when matching
-    const TLV_DISCRIMINATOR: Discriminator = Discriminator::new(INITIALIZE_DISCRIMINATOR);
-}
-/// First 8 bytes of `hash::hashv(&["spl_token_metadata_interface:initialize_account"])`
-const INITIALIZE_DISCRIMINATOR: [u8; Discriminator::LENGTH] = [210, 225, 30, 162, 88, 184, 77, 141];
-// annoying, but needed to perform a match on the value
-const INITIALIZE_DISCRIMINATOR_SLICE: &[u8] = &INITIALIZE_DISCRIMINATOR;
+impl TlvDiscriminator for Initialize {}
```

Comment:
We can just totally remove the `impl TlvDiscriminator for ...` bits since we know they're not necessary, unless you prefer that I do it in a separate PR

Reply:
So just completely eliminate TLV-specific discriminators, and use only the `SPL_DISCRIMINATOR`?

Reply:
Done